mongo/integration/change_stream_test.go

```diff
@@ -447,6 +476,36 @@ func TestChangeStream_ReplicaSet(t *testing.T) {
 			})
 		})
 	})
+	mt.RunOpts("try next", noClientOpts, func(mt *mtest.T) {
+		mt.Run("existing non-empty batch", func(mt *mtest.T) {
+			// If there's already documents in the current batch, TryNext should return true without doing a getMore
+
+			cs, err := mt.Coll.Watch(mtest.Background, mongo.Pipeline{})
+			assert.Nil(mt, err, "Watch error: %v", err)
+			defer closeStream(cs)
+			generateEvents(mt, 5)
+			assert.True(mt, cs.Next(mtest.Background), "expected Next to return true, got false")
```

Comment:
nit: please comment why you're running `Next` in a `TryNext` test.  (presumably, to make sure a batch is retrieved)